### ISSUE TITLE
160 dim for mel spectrogram can also be applied

### ISSUE THREAD

**Issue by begeekmyfriend** [Rayhane-mamah/Tacotron-2#286]
[Here](https://unisound.github.io/end-to-end_tts) is a technical blog article in Chinese written by one of the famous Chinese speech technology company who is doing some researches based on Tacotron. In the picture we can see they are applying 160 dim mel spectrograms for training and prediction. I guess such modification can help the training precision. And I have also tried on my own fork. I found the speed of training is almost not being slow down but the loss is a bit higher (yes, of course). I think it worth trying. 
![image](https://user-images.githubusercontent.com/6031938/49323051-0d5aa300-f551-11e8-83c5-bdf1c3d087be.png)

**Comment by Rayhane-mamah** [comment]
Seems interesting, will check it out!
While we're at it, applying a power of 2 to spectrograms after STFT is also
to keep in mind. I found recent Tacotron papers to do so. It helps taking
off most of the noise in the spectrogram, and inversion gives same quality
(in comoarison to no power) when applying a power of 1/2 (sqrt) during
inversion.

Aside from helping WaveNet quality, that showed some stability in Tacotron
training too and helped stability of the attention mechanism a bit.
However, since more noise is being taken off, the model may overfit quicker
if the data amount is small (6 hours or so).

I hope you find that useful. I will try pushing that as soon as possible.
Thanks for all your hard work.

**Comment by zp75296383** [comment]
I think using power 2 after STFT is the same as changing the LOSS MAE to MSE. Why not use the second option!
